.github/workflows/update-lockfiles.yml

```diff
@@ -0,0 +1,36 @@
+name: Update lockfiles
+
+permissions:
+  contents: write
+  pull-requests: write
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 5 1 * *
+
+jobs:
+  pixi-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
+      - name: Set up pixi
+        uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659 # v0.8.1
+        with:
+          run-install: false
+      - name: Update lockfiles
+        run: |
+          set -euo pipefail
```

Comment:
We really only need `set -o pipefail` here, as GH Actions automatically sets `-e` and this snippet doesn't use any variables that might be affected by `-u`, but I guess it's fine as a fixed expression.